t/helper/test-tool.c

```diff
@@ -57,6 +57,7 @@ static struct test_cmd cmds[] = {
 	{ "sha1-array", cmd__sha1_array },
 	{ "sha256", cmd__sha256 },
 	{ "sigchain", cmd__sigchain },
+	{ "quote_buf_pretty", cmd__quote_buf_pretty },
```

Comment:
for some obscure reason this project likes dashes rather than underscores.
(conservation of shift-keys maybe -- i don't know)
anyway, maybe let the command name be "quote-buf-pretty".